python-sdk/src/astro/table.py

```diff
@@ -129,10 +129,14 @@ def name(self, value: str) -> None:
     @property
     def row_count(self) -> Any:
         """
-        Return the row count of table
+        Return the row count of table.
         """
-        # TODO: Implement this property
-        return 0
+        from sqlalchemy import func, select
+
+        db = create_database(self.conn_id)
+        tb = db.get_sqla_table(table=self)
```

Comment:
You can avoid this step. Directly create the query as "SELECT count(*) from table".

Reply:
query strings are a bit vulnerable to SQL injection

Reply:
But there won't be any input from the user that can manipulate the query string, right? This will be a hard-coded string.

Reply:
right, but still deepsource will complain and will have to using `# skipcq: BAN-B608`